R/helpers.R

```diff
@@ -18,6 +21,8 @@
 
 getRMNDGDPcap <- function(gdpCapfile,
```

Comment:
the raison-d'être of these functions is questionable once we supply GDP as an argument anyways. Maybe one could rename them to `prepareGDP` or even move them as small helper functions directly to the iterative script (if they are not used elsewhere)

Reply:
ok, noted. For now let's include it but let's remember to tackle this.